package.json

```diff
@@ -1,8 +1,4 @@
 {
-  "name": "app-search-firestore-extension",
```

Comment:
All of this detail is only needed if you're publishing an npm package. Since we're not publishing this top level thing, I moved all of it down to `functions/package.json`, which we actually are publishing. I also made this private.